src/App/Utilities/index.js

```diff
@@ -1,9 +1,102 @@
+const utilitiyDocPresenter = [
+    {
+        title: "Border",
+        items: borders,
+        description: <p>Border utility classes.</p>
+    },
+    {
+        title: "Breakpoints",
+        items: breakpoints,
+        description:
+        <>
+            <p>Like Bootstrap our breakpoints are ranged from <CodeTagComponent type="secondary" code="xs" /> to <CodeTagComponent type="secondary" code="xl" />. In addition, we have added yet another breakpoint for even bigger screens called <CodeTagComponent type="secondary" code="xxl" />.</p>
+            <p>Note that when targeting screensize <CodeTagComponent type="secondary" code="xs" />, it is optional to append the infix to the classname.</p>
+        </>
+    },
+    {
+        title: "Color",
+        items: colors,
+        description: <p>We have a few color utility classes. For more information about our colors, refer to this <a href="/identity/color">color section</a>.</p>
+    },
+    {
+        title: "Display",
+        items: displayHidingElements,
+        description: <p>Display utility classes apply to all breakpoints, from <CodeTagComponent type="secondary" code="xs"/> to <CodeTagComponent type="secondary" code="xxl"/>, or you can apply them without breakpoints.</p>
+    },
+    {
+        title: "Flex",
+        items: flex,
+        description: <p>For quickly managing the layout, alignment, sizing of grid columns, navigation, components and more.</p>
+    },
+    {
+        title: "Image",
+        items: images,
+        description: <p>For adding rules to your pictures quickly.</p>
+    },
+    {
+        title: "Sizing",
+        items: sizings,
+        description: <p>Easily make an element as wide or tall as you like. (Relative to it&apos;s parent)</p>
+    },
+    {
+        title: "Spacing",
+        items: spacings,
+        description: <>
+            <p>One can alter only desired sides of the element. Just add the letter t(op), b(ottom), l(eft) or r(ight) after your m(argin) or p(adding).</p>
+            <p>For example, <CodeTagComponent code="mt-5" type="secondary"/> will give <CodeTagComponent code="margin-top: 3rem" type="secondary"/></p>
+            You can also approach the spacings with the x and y-axis. <CodeTagComponent type="secondary" code="my-5"/> will give you <CodeTagComponent code="margin-top: 3rem; margin-bottom: 3rem" type="secondary"/>
+        </>
+    },
+    {
+        title: "Text",
+        items: text,
+        description: <p>For controlling alignment, wrapping, weight and more.</p>
+    },
+    {
+        title: "Visibility",
+        items: visibility,
+        description: <p>Control the visibility without modifying the display of your elements.</p>
+    }
+];
+
+const Utilities = () => {
+    useEffect(() => {
+        expandable.init();
+    }, []);
+
+    return (
+        <div className="doc-container utilities-page">
+            <span className="dg-current-version text-uppercase">Design Guide - v. 10.5.1</span>
```

Comment:
should the version number be modified to be `10.6.0` ? Or is there some hidden magic keeping it in sync?
we probably want to find a way to share the current version number across the project, so we would have it dynamic here. Else it will necessarily get forgotten from time to time (at least by me)